plugins/napari_ADS/_widget.py

```diff
@@ -181,6 +187,16 @@ def __init__(self, napari_viewer):
         fill_axons_button = QPushButton("Fill axons")
         fill_axons_button.clicked.connect(self._on_fill_axons_click)
 
+        remove_axons_button = QPushButton("Axon removal toggle")
+        remove_axons_button.clicked.connect(self._on_remove_axons_click)
+
+        remove_axons_button.setCheckable(True)
+        remove_axons_button.setStyleSheet(
+            "QPushButton:checked{background-color:blue;}"
+        )
+        remove_axons_button.setToolTip("Hold CONTROL/COMMAND and click on an axon to remove it in the axon and myelin masks.")

```

Comment:
Maybe we should also mention in the tooltip that the image layer must be selected? I didn't read the changes in the documentation first and I was a bit confused when I tried this at first.

Reply:
I thought about that yeah, but was concerned with having too long of a message. I have a few other failsafes if the user doesn't do it themselves, but I'll add a short note here too thanks!

Reply:
Resolved in [d9cc40a](https://github.com/axondeepseg/axondeepseg/pull/868/commits/d9cc40a045076ca680933bb1c81bd5b6b286927d)